internal/packager/extract.go

```diff
@@ -118,7 +118,8 @@ func Extract(name string, ops ...func(*types.App) error) (*types.App, error) {
 		}
 		// look for a docker image
 		ops = append(ops, types.WithSource(types.AppSourceImage))
-		return extractImage(name, ops...)
+		app, err := extractImage(name, ops...)
+		return app, errors.Wrap(err, "cannot locate application in filesystem or registry")
```

Comment:
Would it be worth naming the application to be explicit? i.e.

`Cannot locate application 'mike/nosuchapp' in filesystem or registry`?

Either way looks fine 👍